examples/pxScene2d/src/pxText.cpp

```diff
@@ -174,7 +174,7 @@ void pxText::draw() {
   if( getFontResource() != NULL && getFontResource()->isFontLoaded())
   {
     // TODO not very intelligent given scaling
-    if (msx == 1.0 && msy == 1.0 && mCached.getPtr() && mCached->getTexture().getPtr())
+    if (!mDirty && msx == 1.0 && msy == 1.0 && mCached.getPtr() && mCached->getTexture().getPtr())
```

Comment:
Just want to confirm...  Will mDirty only be true for one frame?  After that will mCached contain the updated text fbo?